### PR TITLE
profiles/hydra/master: start hosting https://chaperone.holo.host

### PR DESCRIPTION
**Pull request by zo-el**

### CODE REVIEW

File: overlays/holo-nixpkgs/default.nix [hunk 1]
```diff
@@ -65,6 +72,8 @@ in
   inherit (callPackage cargo-to-nix {}) buildRustPackage cargoToNix;
   inherit (callPackage gitignore {}) gitignoreSource;
 
+  inherit (callPackage chaperone {}) chaperone;
```
Comment (yegortimoshenko) [inline overlays/holo-nixpkgs/default.nix]: This section is alphasorted.

File: overlays/holo-nixpkgs/default.nix [hunk 2]
```diff
@@ -11,6 +11,13 @@ let
     sha256 = "1rcwpaj64fwz1mwvh9ir04a30ssg35ni41ijv9bq942pskagf1gl";
   };
 
+  chaperone = fetchFromGitHub {
```
Comment (yegortimoshenko) [inline overlays/holo-nixpkgs/default.nix]: This too is alphasorted.

File: overlays/holo-nixpkgs/default.nix (hunk 1 above)
Comment (samrose) [reply overlays/holo-nixpkgs/default.nix]: I merged a PR to resolve this

File: overlays/holo-nixpkgs/default.nix (hunk 2 above)
Comment (samrose) [reply overlays/holo-nixpkgs/default.nix]: I merged a PR to resolve this

File: profiles/hydra/master/default.nix [hunk 3]
```diff
@@ -107,6 +107,13 @@ in
       root = pkgs.hpstatus;
       serverName = "hpstatus.holo.host";
     };
+
+    virtualHosts.chaperone = {
```
Comment (yegortimoshenko) [inline profiles/hydra/master/default.nix]: This is also alphasorted. Will fix on merge.

Comment (samrose) [reply profiles/hydra/master/default.nix]: I fixed this in a hotfix PR

File: profiles/hydra/master/default.nix [hunk 4]
```diff
@@ -79,6 +79,21 @@ in
...
     # compatibility reasons.
+
+    virtualHosts.chaperone = {
+      enableACME = true;
+      forceSSL = true;
+      root = pkgs.chaperone;
+      serverName = "chaperone.holotest.dev";
+    };
+
+    virtualHosts.hpstatus = {
```
Comment (yegortimoshenko) [inline profiles/hydra/master/default.nix]: This doesn't seem to belong to this PR?

File: profiles/hydra/master/default.nix [hunk 5]
```diff
@@ -79,6 +79,21 @@ in
     # First HoloPort/HoloPort+ batch points to Hydra-based Nix channel on
     # holoportbuild.holo.host. This has to be left here forever for reverse-
     # compatibility reasons.
+
+    virtualHosts.chaperone = {
+      enableACME = true;
+      forceSSL = true;
+      root = pkgs.chaperone;
+      serverName = "chaperone.holotest.dev";
```
Comment (yegortimoshenko) [inline profiles/hydra/master/default.nix]: Seems to be `chaperone.holo.host` in all other places. Going to update this accordingly.
